accessing_s3_data.ipynb

```diff
@@ -449,7 +468,7 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "get_bbox_group(dc_ls, [70, 98, 20, 60])"
+    "get_bbox_group(dc_ls, bounds=[70, 98, 20, 60])"
```

Comment:
Writing out `bounds=` is nice because then future-you knows what that list is supposed to mean.

You could enforce this by changing the signature to
``` python
def get_bbox_grouop(input_xr, *, bounds=[...])
```

With this you can only set `bounds` by writing out `bounds=`.